### PR TITLE
Drivers: initial cleaning for camera and video drivers

### PR DESCRIPTION
**Pull request by Capri2014**

### CODE REVIEW

File: scripts/camera_and_video.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+# run command_name module_name
+function run() {
+    case $1 in
+        start)
+            start
+            ;;
+        stop)
+            stop
+            ;;
```
Comment (storypku) [inline scripts/camera_and_video.sh:47]: What about adding a `restart` option here?

File: modules/drivers/camera/dag/camera.dag [hunk 2]
```diff
@@ -28,74 +17,6 @@ module_config {
         config_file_path : "/apollo/modules/drivers/camera/conf/camera_front_12mm.pb.txt"
       }
     }
-    components {
```
Comment (michael4338) [inline modules/drivers/camera/dag/camera.dag:31]: why remove this?

File: scripts/camera_and_video.sh (hunk 1 above)
Comment (Capri2014) [reply scripts/camera_and_video.sh:47]: Good suggestion! We may prefer though to create a separate PR to add `restart` options for all related scrips together.

File: modules/drivers/camera/dag/camera.dag (hunk 2 above)
Comment (Capri2014) [reply modules/drivers/camera/dag/camera.dag:31]: 1. No front fisheye installed on vehicle by default. 2. The `camera_compress` are not working as of now so I remove them by default, the `video_compress` is still functional properly under `video/dag/vide.dag`.

**Pull request merged**